defaults-o2-dev-fairroot.sh

```diff
@@ -5,8 +5,6 @@ env:
   CFLAGS: "-fPIC -g -O2"
   CMAKE_BUILD_TYPE: "RELWITHDEBINFO"
 disable:
-  - AliEn-Runtime
-  - AliRoot
```

Comment:
@dberzano why would you remove this, effectively enable the packages? Probably wrong for this default configuration.